lib/PrivateBin.php

```diff
@@ -147,10 +147,7 @@ public function __construct()
                 );
                 break;
             case 'read':
-                // reading paste is disallowed in HTML display
```

Comment:
I think this should stay here. As we use JSON requests for reading the paste, it needs to be only done when reading…
https://github.com/PrivateBin/PrivateBin/pull/216/commits/183ebe518b02c52b22d2ad5f472dbfcb862c83f2#diff-e0c90c7248782346a7f382fd588d1039R150

But ah, you mofved that logic into the `_read` function, okay.